### PR TITLE
Fix colony creation

### PR DESCRIPTION
**Pull request by Thodor12**
Closes [discord report](https://discord.com/channels/139070364159311872/1003402487417548801/1288943006946299936) Colony cannot be created

# Changes proposed in this pull request
- Fix a nullpointer during load because of missing colony center value
- Load all manager classes mid-constructor for consistency
- Remove needless secondary constructor
- Expand remaining constructor with more arguments, and remove logic from loadColony()

## Testing
- [X] Yes I tested this before submitting it.
- [ ] I also did a multiplayer test.

Review please

### CODE REVIEW

File: src/main/java/com/minecolonies/core/colony/Colony.java [hunk 1]
```diff
@@ -119,14 +119,14 @@ public class Colony implements IColony
     private boolean ticketedChunksDirty = true;
 
     /**
-     * List of chunks that have to be be force loaded.
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/core/colony/Colony.java:122]: no

**Review by someaddons — APPROVED** [review]

**Review by Raycoms — APPROVED** [review]
Aside of the one grammar correction, looks okay

File: src/main/java/com/minecolonies/core/colony/Colony.java (hunk 1 above)
Comment (Thodor12) [reply src/main/java/com/minecolonies/core/colony/Colony.java:122]: The doubled up "be" was even indicated by IntelliJ to be gramatically incorrect, so

Comment (Raycoms) [reply src/main/java/com/minecolonies/core/colony/Colony.java:122]: ohhh, I didn't even see it was a double "be", thought you removed the be completely. I think I should go back to bed =D

**Pull request merged**